tools/onnx-subgraph/3rd_files_download.sh

```diff
@@ -0,0 +1,8 @@
+mkdir 3rd
+cd 3rd
+git clone https://github.com/ekg/glia.git
+cp -r glia/json ../include
+cp glia/json-forwards.h ../include
+cp glia/jsoncpp.cpp ../src/lib
```

Comment:
copy files will be listed as unchanged files in git status.
plz do not copy files like this.

I cannot find where `jsoncpp.cpp` file is used.